capture/src/main/kotlin/com/alfresco/capture/CaptureModeSelectorView.kt

```diff
@@ -77,6 +77,13 @@ class CaptureModeSelectorView(
         onMode?.invoke(modes[position])
     }
 
+    fun retainLastState(position: Int) {
```

Comment:
This is a bad API, the control is already mode-aware.
Just create a `setMode` the index can be returned by looking for it in the `modes` list.

Reply:
Done